lib/index.js

```diff
@@ -5,6 +5,8 @@
 
 var path  = require('path'),
     zip   = require('jszip'),
+	sizeOf = require('image-size'),
```

Comment:
Please change from tab to space. I probably have to add `.editorconfig` to not bother with such comments.

Reply:
OK (commit 13e53fb). I change my IDE for use space instead of tab